src/main/java/edu/harvard/iq/dataverse/GuestbookResponse.java

```diff
@@ -65,8 +67,12 @@ public class GuestbookResponse implements Serializable {
     @JoinColumn(nullable=true)
     private AuthenticatedUser authenticatedUser;
 
-    @OneToOne(cascade=CascadeType.ALL,mappedBy="guestbookResponse",fetch = FetchType.LAZY, optional = false)
+    @OneToOne(cascade={CascadeType.REMOVE, CascadeType.MERGE, CascadeType.PERSIST},mappedBy="guestbookResponse",fetch = FetchType.LAZY)
```

Comment:
It seems like we're associating guestbook responses with file downloads even if they are actually access requests. It will cause confusion on the guestbook response via and download were numbers will be inflated

Reply:
Rather than trying to change the queries to check for a filedownload entry, I ~reverted #5863 (which was only done in prep for dealing with guestbook-at-request functionality), added 'AccessRequest' as a new eventtype for gb responses and adapted all of the queries to only count non-AccessRequest entries as downloads. (I also changed the names of the queries dealing with downloads to mention download explicitly. There are other queries that still get the total number of gb entries (i.e. ones related to showing the list of all entries) that stayed as they were.) I also adapted the download estimation query which was just using the statistics about table size and average gb entry size to also use the statistic about how many gb entries have eventtype AccessRequest to maintain an accurate count. (In doing this, I raised the threshold at which the estimate starts getting used - before it would kick in with only a few entries which could make download counts be inaccurate when there were only ~tens of downloads which caused some confusion. )